sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/VirtualMachineExtensionImageOperationsTests.java

```diff
@@ -4,20 +4,23 @@
 package com.azure.resourcemanager.compute;
 
 import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.management.Region;
 import com.azure.resourcemanager.compute.models.OperatingSystemTypes;
 import com.azure.resourcemanager.compute.models.VirtualMachineExtensionImage;
 import com.azure.resourcemanager.compute.models.VirtualMachineExtensionImageType;
 import com.azure.resourcemanager.compute.models.VirtualMachineExtensionImageTypes;
 import com.azure.resourcemanager.compute.models.VirtualMachineExtensionImageVersion;
 import com.azure.resourcemanager.compute.models.VirtualMachineExtensionImageVersions;
 import com.azure.resourcemanager.compute.models.VirtualMachinePublisher;
+import com.azure.resourcemanager.test.utils.PlaybackTimeout;
 import com.azure.resourcemanager.test.utils.TestUtilities;
-import com.azure.core.management.Region;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
 public class VirtualMachineExtensionImageOperationsTests extends ComputeManagementTest {
+
     @Test
+    @PlaybackTimeout(60 * 5)
     public void canListExtensionImages() throws Exception {
```

Comment:
This test contains huge amount of record data. It can take more than a minute to finish. Thus the exception is made.